UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -140,6 +140,26 @@ private async Task<bool> CheckUserState(string userid, string token, ConnectedPl
 			return false;
 		}
 
+		//Do not allow users to log in twice for release servers:
+		if (BuildPreferences.isForRelease)
+		{
+			if (GetByUserID(userid) != null)
+			{
+				if (GetByUserID(userid).Connection != null)
+				{
+					if (playerConn.Connection != GetByUserID(userid).Connection)
+					{
+						StartCoroutine(
```

Comment:
won't it cause problems if someone loses connection but will still be treated as "in-game" so won't be able to join again (at least for a while)?
I think multiplayer games traditionally kick existing player when new one joins

Reply:
Nope, once a player disconnects their NetworkConnection gets destroyed

Reply:
Well then if someone steals your account you won't be able to stop him by joining the game

Reply:
they can reset their password and an admin can kick the player, or wait until next round to get it back